token.go

```diff
@@ -19,38 +22,85 @@ type tokenModule struct {
 }
 
 func (m tokenModule) Register(g *gin.RouterGroup) {
-	tokens := g.Group("/tokens")
-	{
-		tokens.GET("", m.getTokenListHandler)
-		tokens.POST("/search", m.searchTokenListHandler)
-	}
-
 	token := g.Group("/token")
 	{
-		token.GET("/:tokenId", m.getTokenHandler)
+		token.GET("", m.getTokenListHandler)
 		token.POST("", m.createTokenHandler)
-		token.PUT("/:tokenId", m.updateTokenHandler)
+
+		tokenByID := token.Group("/:tokenId")
+		{
+			tokenByID.GET("", m.getTokenHandler)
+			tokenByID.PUT("", m.updateTokenHandler)
+		}
 	}
 }
 
-// getTokenListHandler godoc
+var tokenJSONToColumns = map[string]string{
+	response.TokenJSONFields.TokenID:  database.TokenColumns.TokenID,
+	response.TokenJSONFields.Token:    database.TokenColumns.Token,
+	response.TokenJSONFields.UserName: database.TokenColumns.UserName,
+}
+
+var defaultGetTokensOrderBy = orderby.Column{Name: database.TokenColumns.TokenID, Direction: orderby.Desc}
+
+// getBuildListHandler godoc
 // @id getTokenList
-// @summary Returns first 100 tokens
+// @summary Get slice of tokens.
+// @description List all tokens, or a window of tokens using the `limit` and `offset` query parameters. Allows optional filtering parameters.
+// @description Verbatim filters will match on the entire string used to find exact matches,
+// @description while the matching filters are meant for searches by humans where it tries to find soft matches and is therefore inaccurate by nature.
 // @tags token
-// @success 200 {object} []response.Token
+// @param limit query int false "Number of results to return. No limit if unset or non-positive. Required if `offset` is used." default(100)
```

Comment:
Dupe of previous.

Description here doesn't seem to match the changelog description:
> By default all these new endpoints use a default limit of 100, but this can
  be disabled by specifying `?limit=0`.

Assuming the changelog description is the actual behavior?
```suggestion
// @param limit query int false "Number of results to return. Set to 0 or non-positive to disable. Required if `offset` is used." default(100)
```
Alt.
```suggestion
// @param limit query int false "Number of results to return. Set to 0 to disable. Required if `offset` is used." minimum(0) default(100)
```